tests/Jobs/UpdateWikiEntitiesCountJobTest.php

```diff
@@ -0,0 +1,191 @@
+<?php
+
+namespace Tests\Jobs;
+
+use App\Jobs\UpdateWikiEntitiesCountJob;
+use App\Wiki;
+use Illuminate\Contracts\Queue\Job;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Support\Facades\Bus;
+use Illuminate\Support\Facades\Http;
+use PHPUnit\Framework\MockObject\Exception;
+use Tests\TestCase;
+
+class UpdateWikiEntitiesCountJobTest extends TestCase
+{
+    use RefreshDatabase;
+
+    public function setUp(): void {
+        parent::setUp();
+    }
+
+    public function tearDown(): void {
+        parent::tearDown();
+    }
+
+    public function testEmpty()
+    {
+        Bus::fake();
+        Http::fake();
+
+        Bus::assertNothingDispatched();
+        Http::assertNothingSent();
+    }
```

Comment:
Issue: This test currently doesn't test the job as its `handle` method is never called. See the test below to check how a mock job is being created and its handle method invoked.